### ISSUE TITLE
FIDO metadata failure: Unknown COSE algorithm identifier: -37

### ISSUE THREAD

**Issue by mmoayyed** [Yubico/java-webauthn-server#387]
It appears that the latest FIDO metadata feed has added a new algorithm that cannot be recognized by this library here. Relevant stack trace:

```
Caused by:
com.fasterxml.jackson.databind.exc.ValueInstantiationException: 
Cannot construct instance of `com.yubico.webauthn.data.COSEAlgorithmIdentifier`, problem: Unknown COSE algorithm identifier: -37
at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 220483] 
(through reference chain: com.yubico.fido.metadata.MetadataBLOBPayload$MetadataBLOBPayloadBuilder["entries"]->java.util.HashSet[23]->
com.yubico.fido.metadata.MetadataBLOBPayloadEntry$MetadataBLOBPayloadEntryBuilder["metadataStatement"]->
com.yubico.fido.metadata.MetadataStatement$MetadataStatementBuilder["authenticatorGetInfo"]->
com.yubico.fido.metadata.AuthenticatorGetInfo$AuthenticatorGetInfoBuilder["algorithms"]->java.util.ArrayList[1]->
com.yubico.webauthn.data.PublicKeyCredentialParameters["alg"])
```

**Comment by emlun** [comment]
Fixed in [pre-release `2.5.4-RC1`](https://github.com/Yubico/java-webauthn-server/releases/tag/2.5.4-RC1). We'll promote that to a stable release within the next few days or so if no issues with it come up. Thanks for the report (though this was also detected by the weekly integration test)!

**Comment by mmoayyed** [comment]
Thank you very much for the quick fix.
